app/content/models/minute.py

```diff
@@ -41,10 +58,13 @@ def has_object_read_permission(self, request):
         return self.has_read_permission(request)
 
     def has_object_update_permission(self, request):
-        return self.has_write_permission(request)
+        data = request.data
+        if "group" in data:
+            return check_has_access([data["group"]], request)
+        return False
 
     def has_object_destroy_permission(self, request):
-        return self.has_write_permission(request)
+        return check_has_access([self.group.slug], request)
```

Comment:
Hvorfor sjekker du ikke om group finnes i data? Kan vi anta at group finnes? Vil check_has_access håndtere om group er None, og i så fall, hvorfor sjekker du om group finnes i koden over?

Reply:
Ja hvis du sjekker check_has_access så sjekker filtrerer den etter gruppe slug. Så hvis gruppa ikke finnes så har ikke det noe å si. Men i testene må man man lage gruppa for hver test, siden testene kjører på en tom database. Men har fixtures for dette.